Tests/Extractor/JsExtractorTest.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+namespace Bazinga\JsTranslationBundle\Tests\Extractor;
+
+use PHPUnit_Framework_TestCase;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Translation\MessageCatalogue;
+use Bazinga\Bundle\JsTranslationBundle\Finder\FinderFactory;
+use Bazinga\Bundle\JsTranslationBundle\Filesystem\Filesystem;
+use Bazinga\Bundle\JsTranslationBundle\Extractor\JsExtractor;
+
+final class JsExtractorTest extends PHPUnit_Framework_TestCase
+{
+    const TEST_LOCALE = 'en';
+    const TEST_KEY_1 = 'test-key-1';
+    const TRANSLATION_PATH_PUBLIC = '/translation-path/public';
+    const TRANSLATION_PATH_VIEWS = '/translation-path/views';
+
+    /**
+     * @var JsExtractor
+     */
+    private $sut;
```

Comment:
``sut`` stands for ``subject under testing`` which makes it more clear that this is what we are testing.